.github/workflows/publish-layoutlib.yml

```
@@ -0,0 +1,30 @@
+name: publish-layoutlib
+
+on:
+  workflow_dispatch:
+
+env:
+  GRADLE_OPTS: "-Dorg.gradle.jvmargs=-Xmx4g -Dorg.gradle.daemon=false -Dkotlin.incremental=false"
+
+jobs:
+
+  publish:
+    runs-on: ubuntu-latest
+    if: github.ref =~ 'refs/heads/**/layoutlib**'
```

```suggestion
    if: github.ref =~ 'refs/heads/**/publish-layoutlib**'
```

just looked at my branch name, and thought it'd be useful to add this change so that the push to publish can be discerned from the push to bump its usage in the project.